src/go/rpk/pkg/cli/debug/common/common.go

```diff
@@ -0,0 +1,50 @@
+package common
+
+import (
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+// DebugBundleSharedOptions are the options of a debug bundle that can be
+// shared between a normal and a remote bundle.
+type DebugBundleSharedOptions struct {
+	CPUProfilerWait         time.Duration
+	LogsSince               string
+	LogsUntil               string
+	MetricsInterval         time.Duration
+	MetricsSampleCount      int
+	PartitionFlag           []string
+	Namespace               string
+	LabelSelector           []string
+	LogsSizeLimit           string
+	ControllerLogsSizeLimit string
+}
+
+// InstallFlags installs the debug bundle flags that fills the debug bundle
+// options.
+func (o *DebugBundleSharedOptions) InstallFlags(f *pflag.FlagSet) {
+	f.StringVar(&o.ControllerLogsSizeLimit, "controller-logs-size-limit", "132MB", "The size limit of the controller logs that can be stored in the bundle (e.g. 3MB, 1GiB)")
```

Comment:
```suggestion
	f.StringVar(&o.ControllerLogsSizeLimit, "controller-logs-size-limit", "132MB", "The size limit of the controller logs that can be stored in the bundle. For example: 3MB, 1GiB.")
```

Reply:
Fixed. Except for adding the period as we don't add them to our flags help text which we found was a common pattern in other CLIs